test/rails_app/config/environments/test.rb

```diff
@@ -0,0 +1,13 @@
+Rails.application.configure do
+  config.cache_classes = true
+  config.eager_load = false  
+  config.serve_static_files   = true
+  config.static_cache_control = 'public, max-age=3600'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.